src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.netcoreapp.cs

```diff
@@ -350,6 +351,63 @@ public void UnixDomainSocketEndPoint_InvalidPaths_Throws()
             Assert.Throws<ArgumentOutOfRangeException>(() => new UnixDomainSocketEndPoint(invalidLengthString));
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void UnixDomainSocketEndPoint_RemoteEndPointEqualsBindAddress(bool abstractAddress)
+        {
+            string serverAddress;
+            string clientAddress;
+            if (abstractAddress)
+            {
+                // abstract socket addresses are a Linux feature.
+                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
Can put `[PlatformSpecific(TestPlatforms.AnyUnix)]` attribute, to match the rest of the tests.  (And then like @stephentoub suggests, add appropriate validation for macOS).